src/resolve/resolved-manager.h

```diff
@@ -137,6 +137,9 @@ struct Manager {
         struct stat etc_hosts_stat;
         bool read_etc_hosts;
 
+        /* List of refused DNS Record Types*/
+        Set *refuse_record_types;
```

Comment:
This is not freed in `manager_free()`.

Reply:
@yuwata I thought this is done by `resolved-manager.c` in `manager_new()` with:

```
        _cleanup_(manager_freep) Manager *m = NULL;
````

Reply:
You have to free the set explicitly in manager_free(), so that it is freed whenever the manager is freed.